### PR TITLE
feat: use config map for storing config

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: cmd/api-server/main.go [hunk 1]
```diff
@@ -93,16 +94,35 @@ func main() {
...
-	configRepository := config.NewMongoRespository(db)
+	configRepository := configmongo.NewMongoRespository(db)
+	configMapConfig, err := configmap.NewConfigMapConfig(os.Getenv("APISERVER_CONFIG"), namespace)
+	ui.ExitOnError("Getting config map config", err)
 
+	ctx := context.Background()
 	// try to load from mongo based config first
-	telemetryEnabled, err := configRepository.GetTelemetryEnabled(context.Background())
+	telemetryEnabled, err := configMapConfig.GetTelemetryEnabled(ctx)
 	if err != nil {
```
Comment (exu) [inline cmd/api-server/main.go:104]: will this logic be still applicable? 

if err -> load from envs (fallback)

File: cmd/api-server/main.go [hunk 2]
```diff
@@ -93,16 +94,35 @@ func main() {
...
 		// fallback to envs in case of failure (no record yet, or other error)
 		telemetryEnabled = envs.IsTrue("TESTKUBE_ANALYTICS_ENABLED")
 	}
 
-	clusterId, err := configRepository.GetUniqueClusterId(context.Background())
+	var clusterId string
+	config, err := configMapConfig.Get(ctx)
```
Comment (exu) [inline cmd/api-server/main.go:110]: We still have GetUniqueClusterId method? Why not use it here?

Comment (exu) [reply cmd/api-server/main.go:110]: Ok I see here is just writing to storage in case

**Review by exu — APPROVED** [review]

**Comment by exu** [comment]
one commemnt

File: cmd/api-server/main.go (hunk 1 above)
Comment (vsukhin) [reply cmd/api-server/main.go:104]: Yes, this logic was not changed, we try to read config map, if we can't do it (it doesn't exist), then we get it from env var

File: cmd/api-server/main.go (hunk 2 above)
Comment (vsukhin) [reply cmd/api-server/main.go:110]: yes, we try to read from config map, if we can't get it, then we try to read mongo config, and if we don't have access to mongo config, we generate and save a new  cluster id

**Pull request merged**